libeos-payg/provider.c

```diff
@@ -155,34 +155,41 @@ epg_provider_default_init (EpgProviderInterface *iface)
  * epg_provider_add_code:
  * @self: an #EpgProvider
  * @code_str: code to verify and add
+ * @time_added: return location for the span of seconds added by code_str.
  * @error: return location for a #GError
  *
  * Verify and add the given @code_str. This checks that @code_str is valid, and
  * has not been used already. If so, it will add the time period given in the
  * @code_str to #EpgProvider:expiry-time (or to the current time if
- * #EpgProvider:expiry-time is in the past). If @code_str fails verification or
- * cannot be added, an error will be returned.
+ * #EpgProvider:expiry-time is in the past). On success, @time_added will be
+ * populated by the span of seconds added by @code_str. If @code_str fails
+ * verification or cannot be added, an error will be returned.
  *
  * Calls to this function may be rate limited: if too many attempts are made within
  * a given time period, %EPG_MANAGER_ERROR_TOO_MANY_ATTEMPTS will be returned
  * until that period expires. The rate limiting history is reset on a successful
- * verification of a code.
+ * verification of a code. 
+ *
+ * @time_added is signed because some implementations can reduce the available credit when you add a new code.
+ * the add_code funcionality to actually remove time from the credit available.
```

Comment:
It looks like this last line was accidentally included after re-writing the original sentence.